src/Commands/GitHub/SyncLabels.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+namespace SilverStripe\Cow\Commands\GitHub;
+
+use Exception;
+use Github\Api\Issue\Labels as LabelsApi;
+use SilverStripe\Cow\Commands\Command;
+use SilverStripe\Cow\Utility\SupportedModuleLoader;
+use SilverStripe\Cow\Utility\GitHubApi;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use UnexpectedValueException;
+
+class SyncLabels extends Command
+{
+    protected $name = 'github:synclabels';
+
+    protected $description = 'Syncs GitHub labels to all supported modules';
+
+    /**
+     * @var SupportedModuleLoader
+     */
+    protected $supportedModuleLoader;
+
+    /**
+     * @var GitHubApi
+     */
+    protected $github;
+
+    /**
+     * @param SupportedModuleLoader $moduleLoader
+     * @param GitHubApi $github
+     */
+    public function __construct(SupportedModuleLoader $supportedModuleLoader, GitHubApi $github)
+    {
+        parent::__construct();
+
+        $this->supportedModuleLoader = $supportedModuleLoader;
+        $this->github = $github;
+    }
+
+    protected function configureOptions()
+    {
+        $this->addArgument(
+            'modules',
+            InputArgument::OPTIONAL,
```

Comment:
you can have an array of arguments, I think? Which is a bit nicer than comma delimited

Reply:
TIL